src/diagnostics/OpenPMDWriter.cpp

```diff
@@ -402,11 +410,18 @@ OpenPMDWriter::SetupRealProperties (openPMD::ParticleSpecies& currSpecies,
         std::tie(std::ignore, newRecord) = addedRecords.insert(record_name);
         if( newRecord ) {
             currRecord.setUnitDimension( utils::getUnitDimension(record_name) );
-            currRecord.setAttribute( "macroWeighted", 0u );
-        if( record_name == "momentum" )
-            currRecord.setAttribute( "weightingPower", 1.0 );
-        else
-            currRecord.setAttribute( "weightingPower", 0.0 );
+
+            if( record_name == "weighting") {
+                currRecord.setAttribute( "macroWeighted", 1u );
+            } else {
+                currRecord.setAttribute( "macroWeighted", 0u );
```

Comment:
Do I understand correctly that this changes the behavior for `record_name == "weighting"`, which was incorrect before? Or did I missed sth?